src/tests/test_utils/functional_test_utils/layer_tests_summary/github/skip_configs/CPU/expected_failures_API.csv

```diff
@@ -25,3 +25,4 @@ ov_infer_request_1/OVInferRequestDynamicTests.InferFullyDynamicNetworkWithSetTen
 ov_infer_request_1/OVInferRequestDynamicTests.InferFullyDynamicNetworkWithGetTensor/function=AddTwoOutputEdges_inOutShape=(((1.4.20.20)_(1.4.20.20))((2.4.20.20)_(2.4.20.20)))_targetDevice=CPU_,0
 ov_infer_request/OVInferenceChaining.DynamicOutputToDynamicInput/targetDevice=CPU_,0
 ov_infer_request/OVInferenceChaining.DynamicInputToDynamicOutput/targetDevice=CPU_,0
+ov_plugin_mandatory/OVClassModelTestP.ImportModelWithNullContextThrows/0
```

Comment:
`ov_plugin_mandatory/OVClassModelTestP.ImportModelWithNullContextThrows/0,1`
1 is priority
Please extend this file as it sorted (by alphabet)